training/platform-services/PLATFORM.md

```diff
@@ -0,0 +1,182 @@
+
+More information about Prometheus can be found on their website - https://prometheus.io/docs/introduction/overview/
+
+## Next steps
+
+1. Gain experience and familiarity with the platform services
+2. Please update this file if important information are missing regarding the platform services
+
+Further resources
+----------------------------
+1. Attend the [`Support 101 - Platform issues`][support-101-slides] workshop which gives more detail about the platform services and in particular, issues related to the services
+
+[//]: # (Reference Links and Images)
+
+   [setup-ons-mac]: <https://github.com/ONSdigital/dp/blob/master/guides/GETTING_STARTED.md#set-up-your-mac-for-development-work>
+   [dp-setup-terraform]: <https://github.com/ONSdigital/dp-setup/tree/develop/terraform>
+   [dp-setup-ansible]: <https://github.com/ONSdigital/dp-setup/tree/develop/ansible>
+   [concourse-ons]: <https://concourse.onsdigital.co.uk>
+   [git-pr]: <https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-requests#:~:text=Pull%20requests%20let%20you%20tell,merged%20into%20the%20base%20branch.>
+   [concourse-pipeline]: <https://concourse-ci.org/pipelines.html#pipelines>
+   [docker-image]: <https://jfrog.com/knowledge-base/a-beginners-guide-to-understanding-and-building-docker-images/>
+   [consul-dev-services]: <https://consul.develop.onsdigital.co.uk/ui/eu/services>
+   [consul-dev-nodes]: <https://consul.develop.onsdigital.co.uk/ui/eu/nodes>
+   [nomad-dev-home]: <https://nomad.develop.onsdigital.co.uk>
+   [AWS]: <https://aws.amazon.com/>
+   [nomad-job]: <https://learn.hashicorp.com/tutorials/nomad/get-started-jobs>
+   [nomad-task-group]: <https://www.nomadproject.io/docs/job-specification>
+   [nomad-scheduling]: <https://www.nomadproject.io/docs/internals/scheduling/scheduling>
+   [kibana-dev-home]: <https://kibana.develop.onsdigital.co.uk>
+   [helper-script-dp-setup]: <https://github.com/ONSdigital/dp-setup/tree/develop/scripts>
+   [dp-configs-secrets]: <https://github.com/ONSdigital/dp-configs/tree/master/secrets>
+   [dp-setup-vault-policies]: <https://github.com/ONSdigital/dp-setup/tree/develop/ansible/files/vault-policies>
+   [vault-policy]: <https://www.vaultproject.io/docs/concepts/policies>
+   [florence-vault-policy]: <https://github.com/ONSdigital/dp-setup/blob/develop/ansible/files/vault-policies/florence.hcl>
+   [vault-develop]: <https://vault.develop.onsdigital.co.uk>
+   [grafana-dev-home]: <https://grafana.develop.onsdigital.co.uk>
+   [grafana-dev-nomad-tasks]: <https://grafana.develop.onsdigital.co.uk/d/000000014/nomad-task?refresh=10s&orgId=1>
```

Comment:
I'm not certain these links will always work, the ID might change over time, but we can wait and see if it actually causes any problems because I really dont know. Might be worth asking in case anyone else knows though

Reply:
I had a look online and I couldn't find any information regarding the change of ID (Grafana). But it is possible that the ID can change over time (as it is a logical thing that can happen). I believe that it is worthwhile to keep the links in the documentation to allow easy navigation to different locations. 

In the future, if there is a change in ID, then the best route is to remove the specific links. Either way, if this happens, we can learn from it that the ID can change and it would very minimal impact anyway (just link would direct to a 404)